src/App/App.tsx

```diff
@@ -117,6 +102,14 @@ const ConnectedApp: FC<ConnectedAppProps> = ({
         action: openWindow.openLauncher,
     });
 
+    useEffect(() => {
+        paneName.current = allPanes.map(({ name }) => name);
+    }, [allPanes]);
+
+    useEffect(() => {
+        usageData.sendPageView(paneName.current[currentPane]);
+    }, [currentPane]);
```

Comment:
I don't see the point in adding a ref.
Can't you just do 
```suggestion
    useEffect(() => {
        usageData.sendPageView(allPanes[currentPane].name));
    }, [currentPane]);
```

Reply:
No as this will log access page views when we add the about and the feedback form to the `allPanes`

Reply:
Not sure what you're trying to say.
This useEffect is triggered once on startup with both your or my approach.
No additional useEffect call is run because the `log` or `feedback` panes are added to `allPanes`.

Reply:
I also think the ref `paneName` is not needed.